src/client/pages/OfferNew/Perils/index.tsx

```diff
@@ -43,9 +44,10 @@ export const Perils: React.FC<Props> = ({ offerData }) => {
     [offerData.quotes],
   )
 
-  const handleChangeTab = () => {
+  const handleChangeTab = (name: string) => {
     pushToGTMDataLayer({
-      event: 'offer-tab-click',
+      event: 'perils_tab_click',
+      eventData: { type: changeCase.snakeCase(name) },
```

Comment:
Did @sonnyandersson1 want the data in snack case as well?

Reply:
yeah. he specifically asked for that .